src/SpectatorView.Unity/Assets/SpatialAlignment/README.md

```diff
@@ -10,25 +14,77 @@ Not all spatial alignment strategies support all platforms. See the chart below
+> Note: Best is determined as the lowest index of the supported localizers in it's configured list.
 
-3. Depending on the experience, the SpatialCoordinateSystemManager is directed to create a LocalizationSession for a specific SpatialLocalizer. For experiences with the Spectator role chosen on the SpectatorView component, a SpatialLocalizationInitializer that controls localization is chosen from one of two locations. The experience can choose a preferred list of SpatialLocalizationInitializers on the SpectatorViewSettings prefab. In Unity, the Spectator View / Edit Settings menu option will generate this prefab if it does not exist. On the SpatialLocalizationInitializationSettings component, the PrioritizedInitializers list can specify one or more SpatialLocalizationInitializer components. At runtime, the first SpatialLocalizationInitializer that is supported by both devices will be chosen and used to start spatial alignment. If no app-specific SpatialLocalizationInitializer has been specified on the SpectatorViewSettings prefab, a default set specified by the SpectatorView prefab will be consulted in the same manner. For DSLR filming, the compositor window in the Unity editor tells both the user HoloLens and HoloLens mounted to the DSLR camera when to localize. LocalizationSession logic varies for different spatial alignment strategies. A more in-depth look at each alignment strategy can be found below.
+### Exchange of Localization Information
 
-4. Once the LocalizationSession has completed, a SpatialCoordinate will have been located. The position of this SpatialCoordinate in the local application space is then cached in the SpatialCoordinateSystemParticipant. The SpatialCoordinateSystemParticipant then relays this information to its associated peer device. This allows both devices to know the location of the SpatialCoordinate in each others' local application spaces.
+Having determined the appropriate `SpatialLocalizationInitializer` to use, `SpectatorView` invokes it's `RunLocalization` method.
 
-5. Once the SpatialCoordinate's location and orientation are known for both devices, the spectator device's camera is updated so that the observed content has a physical origin that's at the same location as the user device's local application origin. For mobile devices, this is achieved through the SpatialCoordinateTransformer class that applies an additional transform to the mobile device's tracking camera. For DSLR filming, the editor combines reported head pose information with this SpatialCoordinate information to update its camera.
+1. The `SpatialLocalizationInitializer` is then responsbile to appropriately instantiate and configure a SpatialLocalizer to be used for localization. It must instantiate and configure for both the local (spectator) and remote (User) participants. Example:
```

Comment:
subjective nit:

"The `SpatialLocalizationInitializer` is then responsbile to appropriately instantiate and configure a SpatialLocalizer to be used for localization. It must instantiate and configure for both the local (spectator) and remote (User) participants."
to

"The `SpatialLocalizationInitializer" is then responsible for appropriately instantiating and configuring a SpatialLocalizer for localization. It must instantiate and configure localizers for both the local (spectator) and remote (User) participants."